Documentation/BuildInstructionsLadybird.md

```diff
@@ -82,6 +82,11 @@ sudo apt install qt6-multimedia-dev
 sudo pacman -S --needed autoconf-archive automake base-devel ccache cmake curl libgl nasm ninja qt6-base qt6-multimedia qt6-tools qt6-wayland ttf-liberation tar unzip zip
 ```
 
+Or you can install through AUR :
+```
+yay -S ladybird
+```
+
```

Comment:
The section this is in describes, what prerequisites need to be installed in order to build Ladybird using the scripts inside the repository. In fact, the whole file is about how to get a development environment to start making changes to the codebase.

As such, this mentioning AUR here seems out of place, since installing the AUR package will give you an system-wide installed binary and not a development environment. I feel like this would fit better into the README. But there is no precedent for packaging information in the README either. And official packaging is not planned until the alpha release, so I am not sure if maintainers would feel comfortable with such a mention in the README.